lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -224,6 +227,7 @@ sub _schedule_iso {
     my $onlysame           = delete $args->{_ONLY_OBSOLETE_SAME_BUILD} // 0;
     my $skip_chained_deps  = delete $args->{_SKIP_CHAINED_DEPS}        // 0;
 
+    _handle_distri($args);
```

Comment:
What's this?

Reply:
Likely he wanted to do an in-place conversion in the first place.

Reply:
oops, sorry about that. This belongs elsewhere :D Fixed